RecoHI/HiJetAlgos/python/HiSignalGenJetProducer_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+
+hiSignalGenJets = cms.EDProducer('HiSignalGenJetProducer',
```

Comment:
File name should correspond to the module name: this should then be `hiSignalGenJets_cfi.py`.

In any case, if you don't plan to modify these file any further (they are just two parameters which I don't expect will change even for the coming runs) you can stick on the configuration automatically created by the `fillDescriptions` method.

On the contrary, if you still plan to update these configs, you can keep this explicit python config file, but rather clone the one created by `fillDescriptions`, e.g.
```PYTHON
import FWCore.ParameterSet.Config as cms
from RecoHI.HiJetAlgos.hiSignalGenJetProducer_cfi import hiSignalGenJetProducer
hiSignalGenJets = hiSignalGenJetProducer.clone()
```
(and this file could become a `*_cff` then).

If needed in the future you will modify the parameters in the cloned object if needed